README.md

```diff
@@ -3390,6 +3390,16 @@ Other Style Guides
     - We primarily use [`mocha`](https://www.npmjs.com/package/mocha) at Airbnb. [`tape`](https://www.npmjs.com/package/tape) is also used occasionally for small, separate modules.
     - 100% test coverage is a good goal to strive for, even if it’s not always practical to reach it.
     - Whenever you fix a bug, _write a regression test_. A bug fixed without a regression test is almost certainly going to break again in the future.
+    
+  <a name="testing--is-documentation"></a><a name="30.3"></a>
```

Comment:
please remove this 30.3 link - new numbered sections should not be added; they're only here for legacy reasons.

Reply:
Ok!